docs/installation.rst

```diff
@@ -4,36 +4,46 @@
 Installation
 ============
 
-The following details how to install the ``jdaviz`` Python package.
+.. note::
 
-If you encounter problems while following these installation instructions,
-please consult :ref:`known installation issues <known_issues_installation>`.
-
-You may want to consider installing ``jdaviz`` in a new virtual or conda environment
-to avoid version conflicts with other packages you may have installed.
+    ``jdaviz`` is undergoing constant development. We encourage users to always update
+    to the latest version. In general, it is good practice to install the development
+    version following the instructions below as full released versions may lag behind.
```

Comment:
I am not sure how I feel about this statement, especially given that we strive now to do more frequent releases.

Reply:
I'm happy to change it back once we have more frequent releases.